airflow/migrations/versions/0039_3_0_0_tweak_assetaliasmodel_to_match_asset.py

```diff
@@ -59,7 +59,7 @@ def upgrade():
     """Tweak AssetAliasModel to match AssetModel."""
     with op.batch_alter_table("dataset_alias", schema=None) as batch_op:
         batch_op.alter_column("name", type_=_STRING_COLUMN_TYPE, nullable=False)
-        batch_op.add_column(sa.Column("group", _STRING_COLUMN_TYPE, default=str, nullable=False))
+        batch_op.add_column(sa.Column("group", _STRING_COLUMN_TYPE, default="asset", nullable=False))
```

Comment:
If we're doing this, should we add it to asset model as well?

Reply:
Oh I didn’t even know we didn’t set the default group for asset. I guess either way is fine. Let me think a bit which is better.

Reply:
Currently, we only set the group default in the public interface rather than in the model. I agree that both should work fine, but we might be better off unifying our approach.

Reply:
I decided to do the same for assets. Also I realised the current way we set defaults would not work because `default` is client-side. I fixed this with a three-step (add-fill-constrain) approach for both tables.